functions/bfc-functions.php

```diff
@@ -124,6 +124,121 @@ function bfc_nouveau_has_nav( $args = array() ) {
 	return true;
 }
 
+/* Main Nav Menu */
+function bfc_top_nav_menu_builder() {
+$menuitemarray = array (
+  array (
+    'bfc-topnav-home', // menu item id (and selector name)
+    'menu-item-home menu-item-type-custom menu-item-object-custom current-menu-item current_page_item', // any unique selectors for menu item
+    'home', // parent top nav to highlight
+    '/', // page, use relative urls
+    '/wp-content/themes/bfcom/assets/images/home.svg', // icon          // menu item icon
+    'Home' // menu item text
+  ),
+  array (
+    'bfc-topnav-members',
+    'members menu-item-type-post_type menu-item-object-page',
+    'members',
+    '/members/',
+    '/wp-content/themes/bfcom/assets/images/members.svg',
+    'People'
+  ),
+  array (
+    'bfc-topnav-groups',
+    'groups menu-item-type-post_type menu-item-object-page',
+    'groups',
+    '/groups/',
+    '/wp-content/themes/bfcom/assets/images/groups.svg',
+    'Groups'
+  ),
+  array (
+    'bfc-topnav-resources',
+    'resources menu-item-type-post_type menu-item-object-page',
+    'resources',
+    '/wiki/',
+    '/wp-content/themes/bfcom/assets/images/resources.svg',
+    'Resources'
+  ),
+  array (
+    'bfc-topnav-blogs',
+    'blogs menu-item-type-post_type menu-item-object-page',
+    'blogs',
+    '/sites/',
+    '/wp-content/themes/bfcom/assets/images/blog.svg',
+    'Blogs'
+  ),
+  array (
+    'bfc-topnav-search',
+    'search menu-item-type-post_type menu-item-object-page',
+    'search',
+    '/activity/',
+    '/wp-content/themes/bfcom/assets/images/search.svg',
+    'Search'
+  )
+);
+return $menuitemarray;
+}
+
+function bfc_top_nav() {
+  // Build collection for this template method
+  $menuitemarray = bfc_top_nav_menu_builder();
+
+  // set $topnav to its initial value
+    $topnav = '<ul id="bfc-topmenu" class="medium-horizontal menu">'; //start with the opening ul and its selectors
```

Comment:
Looks like this line might be tabbed in one too far.

Reply:
Fixed.